man/systemd.network.xml

```diff
@@ -749,6 +749,16 @@
           Defaults to false.
           </para>
         </listitem>
+      </varlistentry>
+       <varlistentry>
+        <term><varname>IgnoreCarrierGainLoss=</varname></term>
```

Comment:
`RemainWithoutCarrier=` is more appropriate . Since we are only dealing with the carrier loss same as `setting_mtu`

Reply:
I think IgnoreCarrierGainLoss= is more appropriate because thats what the code does. Ignore changes to carrier gain or loss.

When you say RemainWithoutCarrier= then word itself does not clarify what it does. "Remain what?", "Remain where?"

If developers still insist - I can create PR with the name they want.